@@ -44,6 +44,8 @@
 #include <QtCore/qfloat16.h>
 #include <QtCore/qglobal.h>

+#include <limits>
+
 // include stdlib.h and hope that it defines __GLIBC__ for glibc-based systems
 #include <stdlib.h>
 #include <string.h>

--- old/qtbase/src/corelib/tools/qbytearraymatcher.h
+++ new/qtbase/src/corelib/tools/qbytearraymatcher.h
@@ -42,6 +42,8 @@

 #include <QtCore/qbytearray.h>

+#include <limits>
+
 QT_BEGIN_NAMESPACE



--- old/qtbase/src/tools/moc/generator.cpp
+++ new/qtbase/src/tools/moc/generator.cpp
@@ -37,6 +37,7 @@
 #include <QtCore/qjsonarray.h>
 #include <QtCore/qplugin.h>
 #include <stdio.h>
+#include <limits>
 
 #include <private/qmetaobject_p.h> //for the flags.

